apps-rendering/package.json

```diff
@@ -37,7 +37,7 @@
     "@creditkarma/thrift-server-core": "^0.16.1",
     "@emotion/jest": "^11.10.0",
     "@guardian/apps-rendering-api-models": "^1.1.0",
-    "@guardian/atoms-rendering": "^23.5.0",
+    "@guardian/atoms-rendering": "^23.6.0",
```

Comment:
Just a FYI – commit 44b3b7e201dfee28bfdb5cce5a3ef4e40d8abed7 mentions `32.6` instead of `23.6`.